Resources/Prototypes/_CD/Entities/Clothing/OuterClothing/coats.yml

```diff
@@ -56,8 +56,8 @@
 - type: entity
   parent: ClothingOuterCoatEuropanVet
   id: ClothingOuterCoatEuropanCoalition
-  name: coalition jacket
-  description: A heavy leather jacket, winterized and waterproofed for harsh underwater environments. Emblazoned with a Coalition patch on the shoulders.
+  name: nautical jacket
+  description: A heavy leather jacket, winterized and waterproofed for harsh underwater environments. Emblazoned on the shoulders is a blue compass mounted atop a gear.
```

Comment:
I would just remove all reference to other IPs  even just in passing like this.

Reply:
no colour or outlines mentioned